rust_build_utils/rust_utils_config.py

```diff
@@ -66,11 +66,11 @@
     "windows": {
         "archs": {
             "x86_64": {
-                "rust_target": "x86_64-pc-windows-gnu",
+                "rust_target": "x86_64-pc-windows-msvc",
+            },
+            "aarch64": {
+                "rust_target": "aarch64-pc-windows-msvc",
             },
-        },
-        "env": {
-            "RUSTFLAGS": ([" -C link-arg=-s "], "set"),
```

Comment:
This should also be mentioned in a changelog since some libs can rely on this default being here

Reply:
This is incompatible with msvc compiler